core/includes/common.inc

```diff
@@ -7771,7 +7771,7 @@ function backdrop_common_theme() {
       'variables' => array('markup' => NULL),
     ),
     'table' => array(
-      'variables' => array('header' => NULL, 'rows' => NULL, 'attributes' => array(), 'caption' => NULL, 'colgroups' => array(), 'sticky' => TRUE, 'empty' => ''),
+      'variables' => array('header' => NULL, 'footer' => NULL, 'rows' => NULL, 'attributes' => array(), 'caption' => NULL, 'colgroups' => array(), 'sticky' => TRUE, 'empty' => ''),
```

Comment:
I'd recommend to wrap this long array - actually like it's also done in D7:
```
'variables' => array(
  'header' => NULL,
  'footer' => NULL,
  ...
```
That's so much easier to read and phpcs would nag anyway.

Reply:
@indigoxela it's not just this line; it's dozens of them in this function. I think we should file a follow-up if we want to make that change, and then convert all these arrays.

Reply:
Whatever you decide... :smile: 

It's just that the Drupal commit did that properly - they switched to line-breaks, while your change doesn't.

https://git.drupalcode.org/project/drupal/-/commit/85be7c52bbb57217418830aebb13fc4885d4493e